docs/modules/ROOT/pages/component-prometheus.adoc

```diff
@@ -0,0 +1,21 @@
+= Prometheus
+
+[abstract]
+The Couchbase Prometheus Server can be used to collect cloud-native metrics from Couchbase Servers.
+
+== Overview
+
+https://prometheus.io/[Prometheus^] is a leading open-source monitoring solution. As a https://www.cncf.io/projects/prometheus/[graduated project^] of the Cloud Native Computing Foundation, it has become the industry standard for metrics collection and alert generation on cloud platforms.
```

Comment:
Separate lines please for each sentence then we're good.

Reply:
I wish there was a lint tool which could catch this